src/main/java/com/microfocus/application/automation/tools/run/AutEnvironmentBuilder.java

```diff
@@ -121,13 +121,10 @@ public void log(String message) {
             performer = new AUTEnvironmentBuilderPerformer(autEnvModel, variableResolver, logger);
             performer.start();
             assignOutputValue(build, performer, autEnvModel.getOutputParameter(), logger);
-        } catch (InterruptedException e) {
-            build.setResult(Result.ABORTED);
-            throw e;
-        } catch (Throwable cause) {
+
+        } catch (Exception e) {
```

Comment:
If you catch all type of exceptions, it could swallow ones that are unrelated to your code?